### ISSUE TITLE
Add link to slot: elevation to help people complete this field

### ISSUE THREAD

**Issue by mslarae13** [microbiomedata/nmdc-schema#1053]
Finding elevation based on lat_lon can be difficult. Google maps doesn't provide elevation easily.

Add this link to the slot guidance for elevation
https://www.advancedconverter.com/map-tools/find-altitude-by-coordinates 

This will help users complete this field more easily.

- [x] Update NMDC schema slot: elevation comments
- [x] Update NMDC submission schema slot: elevation comments so the link will appear in the data harmonizer column help
- [ ] Sync subport with new schema

FYI @turbomam , @brynnz22 , @pkalita-lbl

**Comment by brynnz22** [comment]
Comments have been added to `elev` in the nmdc schema.

**Comment by mslarae13** [comment]
So this just needs merged and the subport updated ? (prod, dev, and sandbox)

@pkalita-lbl

**Comment by pkalita-lbl** [comment]
Yeah at some point I can build a new release of `submission-schema` off of the latest `nmdc-schmea` and the bring that into `nmdc-server`. Once that's done the change will be visible in the submission portal on dev. Propagating to sandbox and prod will happen whenever it happens based on the release schedule.

**Comment by mslarae13** [comment]
We'll have submission portal updates for the UF squad, we can just plan to test / make sure all these changes are pushed when we do that update.

**Comment by turbomam** [comment]
## Review

- `elev` is asserted as a Biosample slot in file `src/schema/nmdc.yaml` form the nmdc-schema.
- One part of `make all` is detecting which `Biosample` and `OmicsProcessing` slots are to be imported from MIxS. SInce these terms are imported, we don't generally make any changes to them other than giving them a more structured `range`. But we can assert that they should be used in a cusotmized way int eh context of some class.

Here is the `slot_usage`for `elev` in class `Biosample` in the nmdc-schema.

```yaml
    slot_usage:
      elev:
        title: elevation, meters
        range: float
        comments:
          - All elevations must be reported in meters. Provide the numerical portion only.
          - Please use https://www.advancedconverter.com/map-tools/find-altitude-by-coordinates, if needed, to help estimate the elevation based on latitude and longitude coordinates.
        examples:
          value: 100

```

tl;dr: the "find-altitude-by-coordinates" comment is present in the nmdc-schema

**Comment by turbomam** [comment]
the main branch of submission-schema contains the "" comment, which appears on the basic DataHarmonizer interface now.

I will be making a submission-schema release soon.

<img width="1134" alt="Screen Shot 2023-09-05 at 2 14 01 PM" src="https://github.com/microbiomedata/nmdc-schema/assets/25185273/f007d936-1330-417c-a18f-a0f4904e99cc">

**Comment by turbomam** [comment]
I announced the availability of submisison-schema v7.8.0 in the NMDC Slack's `#general` channel

**Comment by turbomam** [comment]
`https://www.advancedconverter.com/map-tools/find-altitude-by-coordinates` appears as plain text in the DataHarmonizer help, not as a hyperlink.

**Comment by mslarae13** [comment]
@turbomam , some of your comments are confusing me. You keep saying altitude.. this is elevation. It's not the same. 
I don't follow what OmicsProcessing has to do with this?
We have lots of MIxS slots that we implement slot usage to change what GSC provides.


I don't think we should've pushed the submission schema without @pkalita-lbl  present. But it should be fine since it's not on the submission portal.

It appearing as text and not a hyperlink here is probably fine. It should hyperlink in the submission portal. We can test this when we are ready to push this change to the submission portal dev.

**Comment by turbomam** [comment]
@mslarae13, didn't you request that https://www.advancedconverter.com/map-tools/find-altitude-by-coordinates should be suggested as a helpful tool for the `elev` slot?  That's the only context in which I have used the string 'alt' in this issue.

I mentioned `OmicsProcessing` because, like `Biosample`, it is interrogated for which MIxS slots it uses at build time. That means that the MIxS terms they use can't be modified globally. So it's applied as a `slot_usage` for nmdc-schema class `Biosample`, which is currently inherited by all of the biosample-like classes in the submission-schema.

I meant for the mention of `OmicsProcessing to be a learning experience, but I guess it just became a distraction.

I apologize for not consulting @pkalita-lbl or others before making a submission-schema release. Can we please develop a document that clarifies who should be consulted at what stage?

**Comment by mslarae13** [comment]
Yes, sorry. That website says altitude, but it's providing altitude or elevation. The terms are interchanged colloquially, but with respect to the slots we provide and the separating descriptions, it's elevation.  If you click on the link on the left, 'elevation by map' the tool still says altitude. If you're award of a better elevation finding tool, happy to use a different link! This was just the best one I could find. 

I'm still confused though... this isn't high priority. So let's re-visit in October. Maybe discuss at a metadata meeting.
 
<img width="839" alt="Screenshot 2023-09-20 at 5 25 46 PM" src="https://github.com/microbiomedata/nmdc-schema/assets/44581429/832b487a-63b1-4111-8e28-07fc02319303">

**Comment by turbomam** [comment]
What are the "link on the left" and "the tool" that "says altitude"?

Yes, I'm sure we can iron this out in a live meeting.

**Comment by turbomam** [comment]
@pkalita-lbl @mslarae13 this is complete, isn't it?

**Comment by mslarae13** [comment]
Yup! Done!
